clients/linux/src/backend.rs

```diff
@@ -233,6 +234,35 @@ impl LbCore {
         ))
     }
 
+    pub fn import_file(
+        &self,
+        parent: Uuid,
+        destination: String,
+        f: Option<Box<dyn Fn(ImportExportFileInfo)>>,
+    ) -> LbResult<()> {
+        import_file(&self.config, parent, PathBuf::from(destination), f).map_err(map_core_err!(ImportFileError,
+            FileAlreadyExists => uerr_dialog!("A file already exists in the path with the same name."),
+            DocumentTreatedAsFolder => uerr_dialog!("A document is being treated as folder."),
+            ParentDoesNotExist => uerr_dialog!("The folder does not exist."),
+            NoAccount => uerr_dialog!("No account found."),
+            BadPath => uerr_dialog!("An invalid path was used."),
+        ))
+    }
+
+    pub fn set_up_drag_export(
+        &self,
+        parent: Uuid,
+        destination: &str,
+        f: Option<Box<dyn Fn(ImportExportFileInfo)>>,
+    ) -> LbResult<()> {
+        export_file(&self.config, parent, PathBuf::from(destination), f).map_err(map_core_err!(ExportFileError,
+            ParentDoesNotExist => uerr_dialog!("The folder does not exist."),
+            FileAlreadyExistsInDisk => uerr_dialog!("A file already exists in the path with the same name."),
+            NoAccount => uerr_dialog!("No account found."),
+            BadPath => uerr_dialog!("An invalid path was used."),
```

Comment:
Just try to use fewer words like I suggest above. But if you're going to use more words, be very specific and contextual regarding it being export related. You know *how* these errors are going to arise. We should be able to debug if someone tells us just the mere string that came up with the error.